lib/ansible/parsing/mod_args.py

```diff
@@ -179,7 +179,11 @@ def _normalize_parameters(self, thing, action=None, additional_args=None):
             for arg in args:
                 arg = to_text(arg)
                 if arg.startswith('_ansible_'):
-                    raise AnsibleError("invalid parameter specified for action '%s': '%s'" % (action, arg))
+                    err_msg = (
+                        "Invalid parameter specified beginning with keyword '_ansible_' for action '%s': '%s'. "
+                        "Keyword '_ansible_' is reserved and used for the internal engine."
+                    )
+                    raise AnsibleError(err_msg % (action, arg))
```

Comment:
I suggest switching to an f-string here for simplicity:
```suggestion
                    err_msg = (
                        f"Invalid parameter specified beginning with keyword '_ansible_' for action '{action !s}': '{arg !s}'. "
                        "Keyword '_ansible_' is reserved and used for the internal engine."
                    )
                    raise AnsibleError(err_msg)
```